notebooks/geo/geoseries.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
<p>Let's change this to:</p><p> </p><blockquote><strong>Create a GeoSeries from local data</strong></blockquote><blockquote><br></blockquote><blockquote>The GeoSeries constructor accepts local data or a <code>bigframes.pandas.Series</code> object.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1327/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>